### PR TITLE
Checkout testnet scripts from tip to upstream/0.11

### PR DESCRIPTION
**Pull request by danpaul000**
Checkout testnet scripts from tip to upstream/0.11 to enable on-demand behaviors on testnets

### CODE REVIEW

File: ci/testnet-manager.sh [hunk 1]
```diff
@@ -102,79 +130,77 @@ steps:
...
+        ci/testnet-sanity.sh beta-perf-testnet-solana-com ec2 us-west-2b
     )
     ;;
   testnet)
-    # shellcheck disable=2030
-    # shellcheck disable=2031
     (
-      set -ex
-      export NO_LEDGER_VERIFY=1
-      export NO_VALIDATOR_SANITY=1
```
Comment (pgarg66) [inline ci/testnet-manager.sh]: Any reason to not include NO_VALIDATOR_SANITY flag in this deployment?

Comment (mvines) [reply ci/testnet-manager.sh]: imo not worth the risk for v0.11

Comment (danpaul000) [reply ci/testnet-manager.sh]: Should I commit it as shown then?  This file is just an exact copy of the one on master at the moment

Comment (pgarg66) [reply ci/testnet-manager.sh]: I think we should add it back, at least till tesnet is on v11

**Pull request merged**